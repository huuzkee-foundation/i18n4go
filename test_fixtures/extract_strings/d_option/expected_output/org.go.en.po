# filename: d_option/input_files/org/org.go, offset: 715, line: 30, column: 16
msgid "org"
msgstr "org"

# filename: d_option/input_files/org/org.go, offset: 737, line: 31, column: 16
msgid "Show org info"
msgstr "Show org info"

# filename: d_option/input_files/org/org.go, offset: 769, line: 32, column: 16
msgid "CF_NAME org ORG"
msgstr "CF_NAME org ORG"

# filename: d_option/input_files/org/org.go, offset: 979, line: 38, column: 20
msgid "Incorrect Usage"
msgstr "Incorrect Usage"

# filename: d_option/input_files/org/org.go, offset: 1312, line: 54, column: 13
msgid "Getting info for org %s as %s..."
msgstr "Getting info for org %s as %s..."

# filename: d_option/input_files/org/org.go, offset: 1465, line: 59, column: 13
msgid "\n%s:"
msgstr "\n%s:"

# filename: d_option/input_files/org/org.go, offset: 1772, line: 72, column: 26
msgid "%s (%dM memory limit, %d routes, %d services, paid services %s)"
msgstr "%s (%dM memory limit, %d routes, %d services, paid services %s)"

# filename: d_option/input_files/org/org.go, offset: 1976, line: 75, column: 13
msgid "  domains: %s"
msgstr "  domains: %s"

# filename: d_option/input_files/org/org.go, offset: 2060, line: 76, column: 13
msgid "  quota:   %s"
msgstr "  quota:   %s"

# filename: d_option/input_files/org/org.go, offset: 2125, line: 77, column: 13
msgid "  spaces:  %s"
msgstr "  spaces:  %s"

