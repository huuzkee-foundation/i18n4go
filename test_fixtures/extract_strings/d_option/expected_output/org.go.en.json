[{"id":"org","translation":"org"},{"id":"Show org info","translation":"Show org info"},{"id":"CF_NAME org ORG","translation":"CF_NAME org ORG"},{"id":"Incorrect Usage","translation":"Incorrect Usage"},{"id":"Getting info for org %s as %s...","translation":"Getting info for org %s as %s..."},{"id":"\n%s:","translation":"\n%s:"},{"id":"%s (%dM memory limit, %d routes, %d services, paid services %s)","translation":"%s (%dM memory limit, %d routes, %d services, paid services %s)"},{"id":"  domains: %s","translation":"  domains: %s"},{"id":"  quota:   %s","translation":"  quota:   %s"},{"id":"  spaces:  %s","translation":"  spaces:  %s"}]
